cmd/util/flags/cliflags/job.go

```diff
@@ -229,11 +182,10 @@ Matching objects must satisfy all of the specified label constraints.`)
 	cmd.Flags().AddFlagSet(fs)
 	// NB(forrest): don't allow the legacy flag name to be used together with the new flag name.
 	cmd.MarkFlagsMutuallyExclusive("count", "concurrency")
-	cmd.MarkFlagsMutuallyExclusive("labels", "label")
 	cmd.MarkFlagsMutuallyExclusive("selector", "constraints")
 	cmd.MarkFlagsMutuallyExclusive("target", "type")
 	//NB(forrest): we require the name with the type.
 	// TODO (forrest): FOR REVIEW this ais a question for PRODUCT cc Aronchick
 	// - do we want to support type? do we want a dtype flag for each kind of job? --type-ops, --type-daemon, etc?
-	cmd.MarkFlagsRequiredTogether("type", "name")
+	//cmd.MarkFlagsRequiredTogether("type", "name")
```

Comment:
I think this is the only item left to merge the PR. My preference is to decouple the introduction of `--type` flag and enabling different submitting long running job types through `docker run` from this PR as it might take time to convey the problem to product and align on a decision, specially that no one asked for this yet.

Reply:
I have removed the type flag from the command. `target` remains on the command with the same behavior as before.